src/EventExport/SortOrder.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\EventExport;
+
+use CultuurNet\UDB3\Deserializer\MissingValueException;
+
+final class SortOrder
```

Comment:
I think this name is a bit misleading. It is more than just a sort order, it is the sort order of a certain property.